data/projects.js

```diff
@@ -96,6 +96,7 @@ projects = [
         ],
         "difficulty" : "high",
         "issues" : [],
+        "developers involved" :[],
```

Comment:
Yes adding an underscore will be nice, like for collaborating projects field, we have "collaborating_projects".